### PR TITLE
virt-test.virttest.utils_test: bug fix for get_time

### PR DESCRIPTION
**Pull request by ypu**
The output from the commands have some slight change and caused the
function failed. So update the method to get time from related
commands.

### CODE REVIEW

**Comment by ypu** [comment]
ID: 1153896

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

File: virttest/utils_test/__init__.py [hunk 1]
```diff
@@ -204,7 +204,7 @@ def get_time(session, time_command, time_filter_re, time_format):
     :return: A tuple containing the host time and guest time.
     """
     if re.findall("ntpdate|w32tm", time_command):
-        output = session.cmd(time_command)
+        output = session.cmd_output__safe(time_command)
```
Comment (Hao-Liu) [inline virttest/utils_test/__init__.py]: @ypu, There's a duplicated "_" here.

Comment (ypu) [inline virttest/utils_test/__init__.py]: Thanks @Hao-Liu will update it.

**Comment by FengYang** [comment]
WOrks for me

Acked-by: Feng Yang

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

**Pull request merged**